### PR TITLE
Adds blood volume to the Body Scanner & Vitals Monitor

### PR DESCRIPTION
**Pull request by VisVirific**
First of all, thank you Karolis for explaining how to compile VueUI changes so I can actually confirm they work as intended.

* Adds Blood Volume information to Body Scanners, as well as in the printed scan.

**Blood Loss only injury**
blood loss only

**Ruptured Lungs without any blood loss due to coughing blood.**
ruptured lungs no blood coughs

### CODE REVIEW

File: nano/templates/med_diagnostics.tmpl [hunk 1]
```diff
@@ -101,6 +101,10 @@
 		<div class="statusLabel">Blood Oxygenation:</div>
 		<div class="statusValue">{{:helper.round(data.blood_o2)}}%</div>
 	</div>
+ 	<div class="line">
+		<div class="statusLabel">Blood Volume:</div>
+		<div class="statusValue"> {{:data.blood_volume}}%</div>
+	</div>
```
Comment (Geevies) [inline nano/templates/med_diagnostics.tmpl:107]: we should really just delete this file

**Comment by Karolis2011** [comment]
I will love if you created `vueui/uitests/medical/bodyscanner.json` in compliance with ui test JSON format.

**Review by Geevies — APPROVED** [review]

**Comment by VisVirific** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

**Review by Karolis2011 — APPROVED** [review]
Tests look good 
image

**Pull request merged**